src/scripts/main.js

```diff
@@ -353,8 +353,28 @@ const people = [
     slug: 'jacobus-bernardus-van-brussel-1736',
   },
 ];
+// write your code here
+const table = document.querySelector('.dashboard');
```

Comment:
Ensure that there is an HTML element with the class `dashboard` in your document. If this element does not exist, the `querySelector` will return `null`, and attempting to append children to it will result in a runtime error.